templates/cros-ec/cros-ec.jinja2

```diff
@@ -5,15 +5,15 @@
     - repository:
         metadata:
           format: Lava-Test Test Definition 1.0
-          name: cros-ec
-          description: "check /dev/cros_ec"
+          name: {{ plan }}
+          description: "cros-ec test plan"
           os:
-          - oe
```

Comment:
We'll see if this works, but `oe` is actually also used with stripped-down versions of Debian such as this rootfs.  The igt tests are running with a small Debian rootfs and the LAVA job definition uses `oe`.  I guess you've used `debian` in your LAVA jobs so this should probably be fine.

Reply:
Actually this is fine, the `deploy` stage still has the OS set to `oe` which is what I think matters before running tests.